### PR TITLE
move my first application into tutorials page

### PR DESCRIPTION
**Pull request by elmiko**
This change is moving the my first application tutorial to make it more
consistent with the other tutorial content on the site. It has been
observed that even experienced users miss the navbar entry for this
tutorial.

changes
* remove my first app from menu
* change titles in my first app to read better
* add project titles to sparkpi impl docs

### CODE REVIEW

File: my-first-radanalytics-app.adoc [hunk 1]
```diff
@@ -1,10 +1,10 @@
-= My First RADanalytics Application
+= SparkPi, your first Apache Spark application with radanalytics.io
```
Comment (erikerlandson) [inline my-first-radanalytics-app.adoc]: "SparkPi: an introduction to applications using Apache Spark and source-to-image with radanalytics.io"

Is 'radanalytics.io' meant to encompass "oshinko and our tooling" in this blurb?

Comment (elmiko) [reply my-first-radanalytics-app.adoc]: so, i tested out this title but it seems really long on the page when it renders out. how would you feel about "SparkPi: an introduction to application using Apache Spark with radanalytics.io" ?

Comment (erikerlandson) [reply my-first-radanalytics-app.adoc]: that sounds good

**Comment by erikerlandson** [comment]
@elmiko lgtm

**Pull request merged**